package.json

```diff
@@ -40,6 +40,7 @@
     "bluebird": "3.0.5",
     "body-parser": "1.14.2",
     "browserify": "14.0.0",
+    "browserify-istanbul": "^2.0.0",
```

Comment:
Nit: Presuming this is a regular expression, is there a particular reason that this one is anchored at the beginning of the string but most of the others aren't?